bip-update-process.md

```diff
@@ -0,0 +1,708 @@
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the Bitcoin[^capitalization]
+community. The BIP process as defined by BIP 2 thus far seems to have been fashioned to facilitate the design and
+activation of protocol changes. In the past years, BIPs more often describe interoperable features beyond the base
+protocol. The community has had multiple debates about the role of BIP Editors, and some aspects of the process
+specified by BIP 2 that did not seem to achieve the intended goals. This proposal sunsets aspects of the BIP 2 process
+that did not achieve broad adoption, reduces the judgment calls assigned to the BIP Editor role, and delineates the
+BIP Types more clearly.
+
+## Fundamentals
+
+### What is a BIP?
+
+BIPs cover the range of interests of the Bitcoin community. The main topic is technology that supports the Bitcoin
+currency. Most BIPs provide a concise, self-contained, technical description of one new concept, feature, or standard.
+Some BIPs describe processes, implementation guidelines, best practices, incident reports (e.g.
+[BIP 50](bip-0050.mediawiki)), or other information relevant to the Bitcoin community. However, any topics related to
+the Bitcoin protocol, Bitcoin’s peer-to-peer network, and Bitcoin client software may be acceptable.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into implementations. BIPs may be submitted by anyone.
+
+### BIP Ownership
+
+Each BIP is primarily owned by its authors and represents the authors’ opinion or recommendation. The authors are
+expected to foster discussion, address feedback and dissenting opinions, and, if applicable, advance the adoption of
+their proposal within the Bitcoin community.
+
+#### Authors and Shepherds
+
+Authors may want additional support with the BIP process after writing an initial draft. In that case, they may assign
+one or more Shepherds to their BIP. Shepherds are stand in owners of a BIP who were not involved in writing the
+document. They support the authors in advancing the proposal, or act as point-of-contact for the BIP in absence of the
+authors. Shepherds may perform the role of Authors for any aspect of the BIP process unless overruled by an Author.
+Shepherds share ownership of the BIP at the discretion of the Authors.
+
+### What is the significance of BIPs?
+
+BIPs do not define what Bitcoin is: individual BIPs do not represent Bitcoin community consensus or a general
+recommendation for implementation. A BIP represents a personal recommendation by the BIP authors to the Bitcoin
+community. Some BIPs may never be adopted. Some BIPs may be adopted by one or more Bitcoin clients or other related
+software. Some may even end up changing the consensus rules that the Bitcoin ecosystem jointly enforces.
+
+### What is the purpose of the BIP repository?
```

Comment:
Here and throughout this document.

```suggestion
### What is the purpose of the BIPs repository?
```

Reply:
I had just changed that the other way per request of @real-or-random here: https://github.com/murchandamus/bips/pull/2#discussion_r1778753665

Reply:
Thinking more about this, I would say "the Bitcoin Improvement Proposal repository", not "the Bitcoin Improvement Proposal**s** repository", so seems right to me?